### ISSUE TITLE
.sync-exclude.lst directory not found

### ISSUE THREAD

**Issue by begem** [hwittenborn/celeste#85]
I have alltime this error, some time have error ".directory directory not found". I was trying create it manually but it have not reasons.

**Comment by begem** [comment]
Yes, I deleted this file manually, celeste create new file, but error remains (I clear errors in app). Also I was trying create new profile in app, change new local folder.

**Comment by 256shadesofgrey** [comment]
I have the same issue. I deleted that file once, it got recreated, and I was not deleting it again. But every time I start the application and it starts synching, the error reappears, even though the file is present when the application launches.

**Comment by ThunderDanG** [comment]
I also see that error pop up. I haven't deleted that file and I can confirm that it exists. I am using the Flatpak version (0.8.1) on Fedora Silverblue and I am syncing with Proton Drive. If it matters, I have two computers both using Celeste to sync documents stored in Proton at the same time.
If there are any log files you want, let me know.

**Comment by josecastillolema** [comment]
+1 with Google Drive
